### ISSUE TITLE
calendar does not load - missing slash in api url

### ISSUE THREAD

**Issue by mrvnklm** [wdelfuego/nova-calendar#60]
the api route for fetching the events is broken due to a missing slash (using default config and settings)
`The route nova-vendor/wdelfuego/nova-calendarmy-calendar/month could not be found.`

**Comment by wdelfuego** [comment]
That's strange -- the current release is working perfectly for me both in dev and in production. So something is wrong but not in all cases. Reviewing it now.

To get a better idea;

Could you post the contents of `/config/nova-calendar.php`? 

Where does that error appear exactly?

**Comment by wdelfuego** [comment]
Also, are you running Nova under its default `/nova` directory or something custom?

**Comment by mrvnklm** [comment]
running nova on `/`

heres the config which is default
```
<?php

/*
 * © Copyright 2022 · Willem Vervuurt, Studio Delfuego
 * 
 * You can modify, use and distribute this package under one of two licenses:
 * 1. GNU AGPLv3
 * 2. A perpetual, non-revocable and 100% free (as in beer) do-what-you-want 
 *    license that allows both non-commercial and commercial use, under conditions.
 *    See LICENSE.md for details.
 * 
 *    (it boils down to: do what you want as long as you're building and/or
 *     using calendar views, but don't embed this package or a modified version
 *     of it in free or paid-for software libraries and packages aimed at developers).
 */

use App\Providers\CalendarDataProvider;
// use App\Providers\SecondCalendarDataProvider;

return [

    'my-calendar' => [

        /*
         * The class of the calendar data provider for this Nova Calendar
           Don't forget to add the proper use statement above.
           This key is required.
         */
        'dataProvider' => CalendarDataProvider::class,

        /*
         * URI for this Nova Calendar (will be appended to the Nova path, /nova by default)
           This key is required.
         */
        'uri' => 'my-calendar',

        /*
         * Browser window/tab title for this Nova Calendar.
           This key is optional.
           If you remove it or set it to an empty string, the dynamic title displayed above 
           the calendar view will be used as window/tab title in the browser.
         */
        'windowTitle' => 'Nova Calendar',

    ],

    // 'calendar2' => [
    //     'uri' => 'wdelfuego/second-calendar',
    //     'dataProvider' => SecondCalendarDataProvider::class
    //     'windowTitle' => 'Second Calendar',
    // ],

];
```

**Comment by mrvnklm** [comment]
okay so my "fix" seems to be breaking for everything else than `/`

**Comment by wdelfuego** [comment]
No worries, since it's working on my side I'm not blindly accepting the pull request :)
I think it's because you're running Nova from the domain root.

I'll deploy a fix for this today. Thanks a lot for reporting!

**Comment by mrvnklm** [comment]
Alright, thank you!

**Comment by wdelfuego** [comment]
Fixed in 2.0.7 :)

Thanks for not only reporting but even offering a pull request directly :)!
